Mathlib/Analysis/Distribution/ContDiffMapSupportedIn.lean

```diff
@@ -297,6 +297,41 @@ lemma toBoundedContinuousFunctionLM_eq_of_scalars (𝕜' : Type*) [NontriviallyN
     (toBoundedContinuousFunctionLM 𝕜 : 𝓓^{n}_{K}(E, F) → _) = toBoundedContinuousFunctionLM 𝕜' :=
   rfl
 
+open scoped Classical in
+variable (n k K K') in
+/-- If `k ≤ n` and `Q ⊆ Q'`, `monoLM 𝕜 n k Q Q'` is the `𝕜`-linear inclusion of
+`𝓓^{n}_{Q}(E, F)` inside `𝓓^{k}_{Q'}(E, F)`. Otherwise, this is the zero map.
+
+This is in fact continuous (see `monoCLM`). Furthermore:
+* it is a topological embedding when `k = n` and `Q ⊆ Q'` (not in Mathlib yet)
+* it maps bounded sets to compact sets when `k + 1 ≤ n` and `Q ⊆ Q'` (not in Mathlib yet)
+-/
+noncomputable def monoLM :
+    𝓓^{n}_{K}(E, F) →ₗ[𝕜] 𝓓^{k}_{K'}(E, F) where
+  toFun f :=
+    if h : k ≤ n ∧ K ≤ K' then
+      .of_support_subset (f.contDiff.of_le (mod_cast h.1)) (f.support_subset.trans h.2)
+    else 0
+  map_add' f g := by split_ifs <;> ext <;> simp
+  map_smul' c f := by split_ifs <;> ext <;> simp
+
+open scoped Classical in
+@[simp]
+lemma monoLM_apply (f : 𝓓^{n}_{K}(E, F)) :
+    (monoLM 𝕜 n k K K' f : E → F) = if k ≤ n ∧ K ≤ K' then f else 0 := by
+  rw [monoLM]
+  split_ifs <;> rfl
+
+@[simp]
```

Comment:
I understand you want this as a `simp` lemma, but given the shape of the assumption do you imagine it will ever fire?